frontend/.gitignore

```diff
@@ -5,6 +5,9 @@
 /.pnp
 .pnp.js
 
+#Enviornment Variables
```

Comment:
Add a space after '#' and 'V' should be in lowercase

Reply:
Done!